packages/video_player/video_player_android/pigeons/messages.dart

```diff
@@ -69,4 +73,5 @@ abstract class AndroidVideoPlayerApi {
   void seekTo(PositionMessage msg);
   void pause(TextureMessage msg);
   void setMixWithOthers(MixWithOthersMessage msg);
+  void setupAudioSession(AudioSessionMessage msg);
```

Comment:
Why are you plumbing this all the way through the platform channel just to no-op it on the native side? In fact, I'm not clear on why this PR would need to make any changes to the Android implementation at all.